packages/shared-ini-file-loader/src/parseIni.ts

```diff
@@ -15,10 +15,15 @@ export const parseIni = (iniData: string): ParsedIniData => {
         throw new Error(`Found invalid profile name "${currentSection}"`);
       }
     } else if (currentSection) {
-      const item = line.match(/^\s*(.+?)\s*=\s*(.+?)\s*$/);
-      if (item) {
+      const expressionComponents: string[] = line.split("=");
+      const isAssignment: boolean = expressionComponents.length >= 2 && !!expressionComponents[0];
+      if (isAssignment) {
+        const [name, value]: [string, string] = [
+          expressionComponents[0].trim(),
+          expressionComponents.slice(1).join("=").trim(),
+        ];
```

Comment:
Use [indexOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) and [substring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring) instead of split?
```suggestion
      const indexOfEqualsSign = line.indexOf("=");
      if (indexOfEqualsSign !== -1) {
        const [name, value]: [string, string] = [
          line.substring(0, indexOfEqualsSign).trim(),
          line.substring(indexOfEqualsSign + 1).trim(),
        ];
```